lib/charms/opensearch/v0/opensearch_distro.py

```diff
@@ -305,13 +309,79 @@ def write_file(self, path: str, data: str, override: bool = True):
         with open(path, mode="w") as f:
             f.write(data)
 
+    @property
+    def _plugin(self):
+        """Returns the executable path. Should be overloaded, e.g. by the snap command."""
+        return f"{self.paths.home}/bin/opensearch-plugin"
+
+    def add_plugin_without_restart(self, plugin: str, batch: bool = False) -> bool:
+        """Add a plugin to this node. Restart must be managed in separated."""
+        try:
+            args = "install"
+            if batch:
+                args += " --batch"
+            args += f" {plugin}"
```

Comment:
let's just `--batch` for all plugins - given we only offer plugins we trust in conf, and we cannot deal with prompts at the user -> charm level anyway.

Reply:
Okay, doing the same thing for ```add_to_keystore``` and the ```--force```